lib/bundlex/helper/erlang_helper.ex

```diff
@@ -1,8 +1,7 @@
 defmodule Bundlex.Helper.ErlangHelper do
-  @moduledoc """
-  Module containing helper functions that ease determining path to locally-
-  installed Erlang.
-  """
+  @moduledoc false
+  # Module containing helper functions that ease determining path to locally-
+  # installed Erlang.
```

Comment:
Why have you changed this to `@moduledoc false`?

Reply:
It's there only because we need it in Bundlex, I think that this shouldn't be part of API.